src/components/SearchMessages.vue

```diff
@@ -376,6 +355,20 @@ export default {
 			hasFromMeActive: false,
 			startDate: null,
 			endDate: null,
+			dialogButtons: [
+				{
+					label: t('mail', 'Clear'),
+					callback: () => this.resetFilter,
```

Comment:
```suggestion
					callback: () => this.resetFilter(),
```

unless callback expects a curried function I think the close will return the function, not executed

untested